src/main/java/core/basesyntax/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class BalanceHandler implements OperationHandler {
+    private static final String BALANCE = "b";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
```

Comment:
The parameter names `listOfAll` and `listOfFruits` are confusing and do not convey the purpose of these lists. Consider renaming them to reflect their roles more accurately.